helm-charts/basehub/values.jsonnet

```diff
@@ -14,15 +14,20 @@ local emitDaskHubCompatibleConfig(basehubConfig) =
 
 local jupyterhubHomeNFSResources = {
   quotaEnforcer: {
-    path: '/export/%s' % hub_name,
+    config: {
+      QuotaManager: {
+        paths: ['/export/%s' % hub_name],
+        hard_quota: 0,
+      },
+    },
     resources: {
       requests: {
-        cpu: 0.02,
-        memory: '20M',
+        cpu: 0.2,
+        memory: '750M',
       },
       limits: {
-        cpu: 0.04,
-        memory: '30M',
+        cpu: 0.4,
+        memory: '1G',
```

Comment:
This container loops through every dir recursively and sets the quota. It the depth is big and there are lots of user home dirs, I've seen the pod getting OOMkilled.